http/restful-ws/src/main/java/io/cloudevents/http/restful/ws/impl/RestfulWSMessageWriter.java

```diff
@@ -20,67 +20,36 @@
 import io.cloudevents.SpecVersion;
 import io.cloudevents.core.format.EventFormat;
 import io.cloudevents.core.message.MessageWriter;
+import io.cloudevents.http.HttpMessageWriter;
 import io.cloudevents.rw.CloudEventRWException;
 import io.cloudevents.rw.CloudEventWriter;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
What?!

Reply:
Should have been UnsupportedOperationException, I wrote NotImplementedException (because I used that in c# :smile:), then IDE did rest.

Reply:
Anyway this is only for demonstration purposes only in while this is draft.